File: T9. METHODS. DEBUGGING AND TROUBLESHOOTING CODE (Java)/src/P02_SignOfIntegerNumber.java
import java.util.Scanner;

public class P02_SignOfIntegerNumber {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        int inputNumber = Integer.parseInt(in.nextLine());
        PrintSign(inputNumber);
    }
    private static void PrintSign(int number) {

        if (number == 0) {
            System.out.printf("The number %d is zero.", number);
        } else if (number > 0) {
            System.out.printf("The number %d is positive.", number);
        } else {
            System.out.printf("The number %d is negative.", number);
        }
    }
}

File: T9. METHODS. DEBUGGING AND TROUBLESHOOTING CODE/P02.SignOfIntegerNumber.cs
using System;

namespace P02.SignOfIntegerNumber
{
    class Program
    {
        static void Main(string[] args)
        {
            int n = int.Parse(Console.ReadLine());
            PrintSign(n);
        }
        static void PrintSign(int number)
        {
            if (number == 0)
            {
                Console.WriteLine($"The number {number} is zero.");
            }
            else if (number > 0)
            {
                Console.WriteLine($"The number {number} is positive.");
            }
            else
            {
                Console.WriteLine($"The number {number} is negative.");
            }
        }
    }
}